src/components/TodoItem.css

```diff
@@ -0,0 +1,87 @@
+.todo-item {
+  text-align: start;
+  margin-bottom: 10px;
+}
+
+.list-item {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 24px 0;
+}
+
+.todo-text {
+  width: 100%;
+  word-wrap: break-word;
+  position: relative;
+  padding-left: 40px;
+  cursor: pointer;
+  display: inline-block;
+}
+
+.todo-time {
+  color: grey;
+  font-size: 12px;
+  padding-left: 40px;
+}
+
+.remove-button button {
+  background-color: white;
+  border: none;
+  padding-left: 5px;
+}
+
+.remove-icon {
+  font-size: 20px;
+}
+
+.todo-text.completed{
+  text-decoration: line-through;
+  color: grey;
+}
+
+/*Avocado checkbox*/
+.checkbox {
+  display: none; 
+}
+
+.todo-text::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 40px;
+  height: 40px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  animation: avorotate 1s ease-in-out 1;
+}
+
+.checkbox:not(:checked) + label::before {
+  background-image: url("../assets/avocado-whole.svg");
+}
+
+.checkbox:checked + label::before {
+  background-image: url("../assets/avocado-open.svg");
+  animation: avorotate 1s ease-in-out 1;;
+}
+
+
+@keyframes avorotate {
+  0% {
+    transform: rotate(0deg);
+  }
+  25% {
+    transform: rotate(20deg);
+  }
```

Comment:
This time invested on getting the rotation to work, it paid off, it looks amazing in the end result 💯 
-Linda&Etna